.github/workflows/Attach Artifacts.yml

```diff
@@ -39,11 +45,11 @@ jobs:
       - name: Create Bundle
         run: |
           tar -C $(pwd)/package --owner=0 --group=0 --format=posix -czvf homebridge-config-ui-x-${{ github.event.inputs.tag }}.tar.gz .
-          shasum -a 256 homebridge-config-ui-x-${{ github.event.inputs.tag }}.tar.gz > SHASUMS256.txt
+          shasum -a 256 homebridge-config-ui-x-${{ needs.analyze-tags.outputs.previous-tag }}.tar.gz > SHASUMS256.txt
```

Comment:
isn't github.event.inputs.tag missing here ?

Reply:
Yes, I will get that added.